content/2-running-automation-testing/guide.md

```diff
@@ -0,0 +1,202 @@
+# 2. Running automation test with Kobiton
+
+## 2.1 Prerequisites
+
+* Username 
+1. Go to portal.kobiton.com
+2. In the upper right hand corner, click on your name and in the drop down menu, click on Profile. 
+
+![profile](../../images/2-profile.png)
+
+3. You should see the username. 
+
+![username](../../images/2-user.png)
+
+* API key
```

Comment:
Same comment for the layout, should make the bullet look more stronger (use `####` or something like that)